### DISCUSSION TITLE
How to run javascript if your district blocked it?

### DISCUSSION THREAD

**Discussion by Endy3838** [3kh0/ext-remover#976] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there any way I  can run javascript because it seems like my school blocked javascript since they have the following commands blocked:
 "JavaScript://*",
  "chrome://kill/",
  "chrome://hang/"
]

**Comment by nathblade16** [comment]
Do you have ublock origin unblocked?
if so, you can use urun or ublock

Are you on a version below 121?
if so you can use skiovox breakout(from the titanium network discord) and shovel(also from the titanium network disocrd)

Are you on a version 121 or higher
if so you can downgrade to 120 or even 119

**Reply by v4pl** [reply]
What does the Skiovox breakout even do? I use Skiovox and can run JavaScript

**Reply by nathblade16** [reply]
I use skiovox breakout, it is basically global xss on extensions and permanent, it allows you to inject code into extensions on your normal profile not your skiovox profile

**Reply by v4pl** [reply]
Ok tysm

**Reply by v4pl** [reply]
Also join our discord sever https://discord.gg/MH3mcPDg2h

**Reply by nathblade16** [reply]
I will do it when I get home

**Reply by Endy3838** [reply]
I am on version  120.0.6099.302 so I suppose I can try downgrading

**Reply by AshtonDavies** [reply]
> I am on version  120.0.6099.302 so I suppose I can try downgrading
> 

Hopefully you're aware that you can't downgrade the regular way from v120, and that the method for versions later than v115 can brick the Chromebook (based only on @vaperbest 's experience with that method).

**Reply by v4pl** [reply]
Fr my Chromebook got bricked so don't try

**Reply by v4pl** [reply]
Join our discord server for help https://discord.gg/VQr7E7D7CV

**Reply by Endy3838** [reply]
alr then thanks for telling me

**Reply by Endy3838** [reply]
so there's no other way to downgrade?

**Reply by AshtonDavies** [reply]
> so there's no other way to downgrade?
> 

Not that I'm aware.

**Reply by Endy3838** [reply]
wallahi im finished

**Comment by xoidex** [comment]
just use a website like https://jsconsole.com/
or downgrade
if you're trying to run a bookmarklet https://blobby-boi.github.io/BlobeBM/ might work (it depends on what the bookmarklet does)
other than that you're screwed

**Comment by AshtonDavies** [comment, marked as answer]
For goodness sake, there are over seven other discussions related to bookmarklets being disabled!

How can you find that `javascript://` URLs are on your blocklist but completely ignore GitHub's simple directions to check for existing discussion topics, which takes just one click?
